src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+public class Money {
+    public static final int MONEY_UNIT = 1000;
+    public static final int MINIMUM_MONEY = 1000;
+
+    private int money;
```

Comment:
```suggestion
    private final int money;
```
`Money`는 불변이어야할 것 같네요 :) `LottoNumber`와는 다르게 vo로 설계하지 않으셨는데 vo에 대한 기준을 맞추어나가시는 것도 많이 도움되실 것 같아요. 개인적으론 vo들은 일관성있는 형태를 띄는 것이 좋다고 생각합니다.

Reply:
이 부분에서 질문이있습니다..!
제가 LottoNumber 를 VO 로 설계하고 Money 를 VO 로 설계하지 않았다고 하셨는데
정확히 VO 가 되려면 충족해야 하는 조건이 있는건가요?

Reply:
`Money`가 돈으로써 vo의 의미를 가지고 있고, `Money(1000)`은 객체의 참조값이 달라도 1000원이라는 그 의미가 같아야해요. 그래서 vo가 되려면 equals hashCode를 통해 동일한 값을 지니는 객체에 대해서 동등성을 보장해주는 것이 하나의 조건이 될 수 있을 것 같아요 :)